Chart issue Fixed

srinivasaraok12 opened the pull request:
**1)what went well?**
implemented state management using Ngrx implemented multiple projects under same applications
implemented facade pattern it's acts the interface in this architecture shared components used
multiple applications maintains is easy
2)what would you change?**
-this application loading time is more,implement universal is added advantage
move all static data into separate .ts files
3)**code smells assign ?**
-wrong variable in assign to chart component,so data is not displayed
-unsubscribe the observeble to avoid memeory leakages

kandarp-patel51 commented:
> **1)what went well?**
> implemented state management using Ngrx implemented multiple projects under same applications
> implemented facade pattern it's acts the interface in this architecture shared components used
> multiple applications maintains is easy
> 2)what would you change?**
> -this application loading time is more,implement universal is added advantage
> move all static data into separate .ts files
> 3)**code smells assign ?**
> -wrong variable in assign to chart component,so data is not displayed
> -unsubscribe the observeble to avoid memeory leakages

In Task 1, if you see the description, they want us to implement the things which we are highlighting in 2 and 3 question. You have provided review on 2 question but didn't implement it. You should do it.

srinivasaraok12 commented:
1)what went well?
implemented state management using Ngrx implemented multiple projects under same applications
implemented facade pattern it's acts the interface in this architecture shared components used
multiple applications maintains is easy
2)what would you change?**
move all static data into separate .ts files
3)code smells assign ?
-wrong variable in assign to chart component,so data is not displayed
-unsubscribe the observeble to avoid memeory leakages
